src/Workspaces/CSharp/Portable/Formatting/Rules/ElasticTriviaFormattingRule.cs

```diff
@@ -444,7 +446,7 @@ private bool IsWhitespace(SyntaxTrivia trivia)
                 || trivia.Kind() == SyntaxKind.EndOfLineTrivia;
         }
 
-        private int GetNumberOfLines(SyntaxTriviaList triviaList)
+        private int GetNumberOfLines(IEnumerable<SyntaxTrivia> triviaList)
```

Comment:
I almost asked why make this change when you now force boxing the `SyntaxTriviaList`, but then I looked at the implementation and realized that it probably doesn't matter...